scss/_variables.scss

```diff
@@ -282,7 +282,10 @@ $nested-kbd-font-weight:      $font-weight-bold !default;
 
 $list-inline-padding:         5px !default;
 
-$mark-bg: #fcf8e3 !default;
+$mark-bg:                     #fcf8e3 !default;
+
+$hr-margin-top:               1rem !default;
```

Comment:
Let's reassign this from `$spacer` and knock down to a single value to match our `x`/`y` approach elsewhere.

```
$hr-margin-y: $spacer !default;`
```

Reply:
👍